apps/feedback/serializers/validators.go

```diff
@@ -0,0 +1,90 @@
+package serializers
+
+import (
+	"reflect"
+	"regexp"
+	"time"
+
+	mapset "github.com/deckarep/golang-set"
+
+	"github.com/jinzhu/gorm"
+
+	feedbackModels "github.com/iReflect/reflect-app/apps/feedback/models"
+	validator "gopkg.in/go-playground/validator.v8"
+)
+
+// Regex to test the response format of a question
+const questionResponseRegexString = "^[0-9]+(,[0-9]+)*$"
```

Comment:
see what is the right syntax for naming const

Reply:
Done